cypress/integration/011-renewal-registration.js

```diff
@@ -0,0 +1,7 @@
+describe('Renewal registration page ', function () {
+  beforeEach(() => {
+    cy.visit('/renewal-registration');
+    cy.get('#main-content form button.naturescot-forward-button').click();
+    cy.url().should('include', '/renewal-postcode');
```

Comment:
Pages need to be visited in the correct order. In the `beforeEach` you need to visit and move through all the pages that are required before this point (e.g. go to the `/renewal-intro` page first, then visit this page.

Also you should probably put a test (outside of the `beforeEach`) that tests the `<h1>` is correct, rather than the URL. The URL will change on a visit regardless of whether the page renders correctly or not.

This applies to the other tests too.